run_dir/static/css/status_b5.css

```diff
@@ -84,3 +84,28 @@ input.quantity_updateable {
     position: relative;
     overflow-y: scroll;
 }
+
+/* Make <samp> (flow cell IDs usually) a little smaller */
+samp { font-size:0.8em; }
+
+/*Tables*/
+th { background-color: #ddd!important; }
```

Comment:
What do you think about making these rules for the table slightly more specific, for example for a certain table class which we then can add to the flowcells, I feel that it might not be suitable for all tables?

Reply:
Added a new style that has to be added to make the header dark.